### PR TITLE
Check time sync in prereqs

### PR DESCRIPTION
**Pull request by lablans**

### CODE REVIEW

**Comment by lablans** [comment]
@torbrenner , see also https://www.freedesktop.org/software/systemd/man/systemd-time-wait-sync.service.html

**Comment by lablans** [comment]
@torbrenner This is now ready to sync, please check.

File: lib/prerequisites.sh [hunk 1]
```diff
@@ -65,6 +65,35 @@ if [ -e /etc/bridgehead/vault.conf ]; then
...
+SERVERTIME="$(https_proxy=$HTTPS_PROXY_URL curl -m 5 -s -v $BROKER_URL 2>&1)"
+RET=$?
+set -e
+if [ $RET -ne 0 ]; then
+	log WARN "Unable to connect to Samply.Beam broker at $BROKER_URL. Please check your proxy settings.\nThe currently configured proxy was \"$HTTPS_PROXY_URL\". This error is normal when using proxy authentication."
+	log WARN "Unable to check clock skew due to previous error."
+else
+	log INFO "Checking clock skew ..."
+
+	SERVERTIME=$(echo -e "$SERVERTIME" | grep Date | sed -e 's/< Date: //')
```
Comment (torbrenner) [inline lib/prerequisites.sh:83]: In my environment (WSL, curl version 7.68.0) the response saved in "SERVERTIME" doesn't contain "Date". It instead contains multiple instances of "date" (see below). With this, the SERVERTIME variable will be empty following this line.

*  start date: Sep 27 03:39:57 2022 GMT
*  expire date: Dec 26 03:39:56 2022 GMT
< date: Tue, 04 Oct 2022 05:55:13 GMT

Comment (lablans) [reply lib/prerequisites.sh:83]: Fixed in 9facafd0c4939e2d6b7fab6a83a9b6593223f139

**Pull request closed without merging**